packages/code-studio/package.json

```diff
@@ -32,6 +32,7 @@
     "cross-env": "^7.0.2",
     "deep-equal": "^2.0.4",
     "fira": "github:mozilla/fira#4.202",
+    "http-proxy-middleware": "^2.0.1",
```

Comment:
dev-dependencies

Reply:
The instructions do specify it as a dependency...: https://create-react-app.dev/docs/proxying-api-requests-in-development/

I'll try

Reply:
Looks like it worked, good call

Reply:
CI did not like that 🤷, sorry.